javav2/example_code/elasticbeanstalk/src/main/java/com/aws/example/CreateApplication.java

```diff
@@ -0,0 +1,77 @@
+//snippet-sourcedescription:[CreateApplication.java demonstrates how to create an AWS Elastic Beanstalk application .]
+//snippet-keyword:[SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-service:[AWS Elastic Beanstalk ]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/10/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.aws.example;
+
+//snippet-start:[eb.java2.create_app.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.elasticbeanstalk.ElasticBeanstalkClient;
+import software.amazon.awssdk.services.elasticbeanstalk.model.CreateApplicationResponse;
+import software.amazon.awssdk.services.elasticbeanstalk.model.CreateApplicationRequest;
+import software.amazon.awssdk.services.elasticbeanstalk.model.ElasticBeanstalkException;
+//snippet-end:[eb.java2.create_app.import]
+
+/**
+ * To run this Java V2 code example, ensure that you have set up your development environment, including your credentials.

```

Comment:
Before running this Java V2 code example, set up your development environment, including your credentials.